todolist copy 3/main.js

```diff
@@ -0,0 +1,74 @@
+const addTaskButton = document.getElementById('addTask');
+const deskTaskInput = document.getElementById('taskInput');
+const toDoList = document.getElementById('tasklist');
+const clearing = document.getElementById('clearAll');
+
+
+let tasks = [];
+!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks'));
+
+function Task(description) {
+	this.description = description;
+	this.completed = false
+}
+
+let toDoElem = [];
+
+const createTask = (task, index) => {

```

Comment:
this string of HTML code is hard to read, let's split it into a few functions createCheckbox(), createDeleteButton(), createText() using DOM methods like createElement(), setAttribute()